deploy/csi-azuredisk-node-windows.yaml

```diff
@@ -0,0 +1,151 @@
+---
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: csi-azuredisk-node-win
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      app: csi-azuredisk-node-win
+  template:
+    metadata:
+      labels:
+        app: csi-azuredisk-node-win
+    spec:
+      nodeSelector:
+        beta.kubernetes.io/os: windows
+      priorityClassName: system-node-critical
+      containers:
+        - name: liveness-probe
+          imagePullPolicy: Always
+          volumeMounts:
+            - mountPath: C:\csi
+              name: plugin-dir
+          image: mcr.microsoft.com/oss/kubernetes-csi/livenessprobe:master
```

Comment:
we should use a specific version instead of `master`, otherwise it may break in the future.
do you know what's the exact versions?